src/assets/colors.tsx

```diff
@@ -0,0 +1,14 @@
+export default {
+  gray: "#f8f8f8",
+  grayRegular: "#e5e5e5",
+  grayMedium: "#cccccc",
+  graySemibold: "#666666",
+  grayBold: "#333333",
+  purpleLight: "#e2d2ff",
+  purpleRegular: "#6e1fff",
+  redRegular: "#f7625a",
+  yellowRegular: "#fdefd1",
+  yellowMedium: "#fce1a8",
+  yellowBold: "#f1af1b",
+  greenRegular: "#0eda83"
+}
```

Comment:
They're in JSON because Stylus [supports importing that](https://github.com/artsy/elan/blob/master/components/lib/index.styl#L4), so it might take some re-jiggering to get ES6 modules to work with the rest of Elan.

FWIW I support some copy pasting and not trying to reuse Elan if it makes life easier. I think using React and Javascript as the lower common denominator to share UI among projects will be the sanest path forward and I'm happy to completely abbandon Stylus/Jade/Coffeescript/etc. to make marching towards that easier.